lms/templates/main.html

```diff
@@ -153,7 +153,7 @@
       <%
         if theme_enabled() and not is_microsite():
           footer_file = 'theme-footer.html'
-        elif settings.FEATURES['IS_EDX_DOMAIN'] and not is_microsite():
+        elif settings.FEATURES.get('IS_EDX_DOMAIN', False) and not is_microsite():
```

Comment:
This is the way we get values out of settings.FEATURES throughout the platform, out of an abundance of caution. If someone deletes the variable out of common.py, things won't weirdly fail.